modules/virt-using-supported-boot-source-names.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-using-supported-boot-source-names_{context}"]
+= Supported boot source names
+
+When you add boot sources, you associate them with templates for specific operating systems.
+
+It is important that the boot source volume name matches the predefined name of the template. If the volume name does not match the name defined by the template, you will be unable to add the boot source and use it to create virtual machines.
+
+If you add a boot source by using the web console, you do not need to supply the boot source name because it is already associated with the appropriate template. When using the CLI, you must specify the supported boot source name in your YAML code when creating a volume.
```

Comment:
` When using the CLI,` wasn't CLI removed from the scope of this doc? if not the namespace should be included as well

Reply:
CLI is still out for 2.6. This module is commented from appearing in the 2.6 build. I did not remove it from the repo as we will need it when doc'ing CLI in 2.6.1. See Test Build Link above. Thanks.